src/plugins/media-analysis/media-analysis.less

```diff
@@ -41,30 +42,69 @@
   }
 }
 
-.MediaAnalysisDisplay-chapterTexts {
-  height: 35px;
+.MediaAnalysisDisplay-chapterPointers {
   display: flex;
-  overflow: hidden;
+  height: calc(@media-analysis-pointer-border-width * 2);
+}
+
+.MediaAnalysisDisplay-chapterPointer {
+  position: relative;
+
+  &:hover {
+    cursor: pointer;
+    background-color: @edu-background-color-medium;
+  }
+
+  &:hover,
+  &.is-visible {
+    &::after {
+      content: '';
+      position: absolute;
+      top: @media-analysis-pointer-border-width;
+      left: calc(50% - @media-analysis-pointer-border-width);
+      width: 0;
+      height: 0;
+      border-left: @media-analysis-pointer-border-width solid transparent;
+      border-right: @media-analysis-pointer-border-width solid transparent;
+      border-bottom: @media-analysis-pointer-border-width solid @edu-text-color-secondary;
+    }
+  }
+}
+
+.MediaAnalysisDisplay-viewedChapterInfoWrapper {
+  display: flex;
+  flex-direction: column;
   padding-bottom: 5px;
+  padding: @media-analysis-text-padding;
+  border: 1px solid @edu-border-color-light;
+  overflow: hidden;
+}
+
+.MediaAnalysisDisplay-viewedChapterInfo {
+  overflow: hidden;
 
   &.is-expanded {
-    height: unset;
+    overflow-y: auto;
   }
 }
 
-.MediaAnalysisDisplay-chapterText {
-  padding: @media-analysis-text-padding;
-  border-right: 1px dotted @edu-border-color-base;
+.MediaAnalysisDisplay-viewedChapterInfoTitle {
+  text-align: center;
+  .m-overflow-ellipsis;
+}
```

Comment:
It would be nice to add some margin on the bottom here. If the annotation contains markdown with multiple paragraphs then this title will be closer to the first paragraph than the second paragraph which looks a little bit off. Also, what do you think of making this title bold so to differentiate it even more from the annotation text?

Reply:
Good suggestions, I've done the changes and also added a pointer on the bottom of the chapter text box, to correlate the text with the player progress bar segments.

Reply:
Nice!